### ISSUE TITLE
Support for [missing root squash option in Azure NetApp Files volume creation]

### ISSUE THREAD

**Issue by y-halim** [terraform-providers/terraform-provider-azurerm#9943]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The root squash option is available on the portal (Root Access = On or Off) but it is missing under azurerm_netapp_volume  Terraform resource: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/netapp_volume.

### New or Affected Resource(s)

* azurerm_netapp

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by kaneuffe** [comment]
According to the documentation [here](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/netapp_volume#export_policy_rule) there is a new attribute allow_root_access. Unfortunately terraform plan does not recognize it:

Error: Unsupported argument

  on create_anf.tf line 37, in resource "azurerm_netapp_volume" "microhack_anf_volume":
  37:     allow_root_access = true

An argument named "allow_root_access" is not expected here.

**Comment by segraef** [comment]
Doco seems to be outdated and needs to be adjusted as per changelog https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/CHANGELOG.md#2550-april-08-2021

As of azurerm@2.55 the right argument would be

`root_access_enabled = true`

**Comment by paulomarquesc** [comment]
I was looking at the Azure NetApp Files issues backlog and looks like this one can be closed since it was resolved by #11386 PR.

**Comment by katbyte** [comment]
closing as requested by the service team, let us know if its still a problem!
